browser/js/common/directives/navbar/navbar.html

```diff
@@ -106,9 +98,9 @@
 	<div ng-class='class'>
 		<div class='dropdown-lists'>
 
-			<div class='dropdown'>
-				<div class='dropdown-title' ng-click='goToSection(".overview", "home")'>What is Kinetic</div>
-				<div class='dropdown-content'>
+			<div class='dropdown' ng-init='whatIs = true'>
```

Comment:
This can just be on the scope in the controller -- you don't need to set this inline.